mage_ai/frontend/oracle/elements/Button/index.tsx

```diff
@@ -72,6 +74,15 @@ const ButtonStyle = styled.button<ButtonProps>`
     border-right: none;
   `}
 
+  ${props => props.negative && `
+    background-color: ${light.background.negative}; !important
+  `}
+
+  // TODO: Change to Lime 40 theme.
+  ${props => props.positive && `
+    background-color: ${light.background.positive}; !important
+  `}
+
```

Comment:
i wonder if we should just add a `background-color` prop

Reply:
I had mentioned to @nathaniel-mage and @shrey-mage to use props on the components for theming in case we add a dark theme later. Although it shud be used like `{(props.theme.background || light.background).positive};` instead of a hardcoded color.

Reply:
Also not sure if that `!important` is necessary

Reply:
> Arrow icon looks slightly large. Figma designs have it at 10x9. Perhaps size of 10 works better.
<img width="192" alt="image" src="https://user-images.githubusercontent.com/90282975/173463940-ce69fcf5-dd01-4c64-afab-2a792215fafa.png">

Size 10 was actually smaller. I set the Text to large to fit the shape better.

Here is Size 10 on a normal text
<img width="208" alt="image" src="https://user-images.githubusercontent.com/90282975/173463898-d13ad8cd-533a-4bb3-8119-5419148aee8f.png">

Which do you think fits better? Or should I look for a different combination?

Reply:
For this one, I wasn't sure if we needed to add the props.theme... since everything else in Button doesn't have it. I'll add it as a story to clean up all the button styles.

Reply:
@nathaniel-mage Just because it was done that way for the other props doesn't necessarily mean it was the right way to do it.